src/db/seeders/trackgroup-seeder.js

```diff
@@ -37,8 +37,9 @@ module.exports = {
           name: 'artist'
         }
       })
+      //  FIXME: 'users' is replicated in users-seeder.js
       await queryInterface.bulkInsert('users', [{
-        id: faker.datatype.uuid(),
+        id: '251c01f6-7293-45f6-b8cd-242bdd76cd0d',
```

Comment:
This will break because it's the same id for the `listener` and the `artist` user.

Reply:
this file shouldn't even be here. Not sure what happened, but clearly a mistake on my part.